packages/ssz/src/type/containerNodeStruct.ts

```diff
@@ -77,9 +77,6 @@ export class ContainerNodeStructType<Fields extends Record<string, Type<unknown>
 
   // ContainerNodeStructType can only parse proofs that contain all the data.
   // TODO: Support converting a partial tree to a partial value
-  getPropertyGindex(): null {
-    return null;
-  }
```

Comment:
It's been a while since I dealt with this code, but what about the comment?

> ContainerNodeStructType can only parse proofs that contain all the data

Reply:
I agree with the comment. I can put in a more effort to make the UX a bit nicer here